apps/condo/domains/acquiring/schema/RegisterMultiPaymentForInvoicesService.test.js

```diff
@@ -336,6 +343,72 @@ describe('RegisterMultiPaymentForInvoicesService', () => {
                 }),
             ]))
         })
+
+        test('register multiPayment for invoice from different organization', async () => {
+            const utils1 = new TestUtils([ResidentTestMixin])
+            const utils2 = new TestUtils([ResidentTestMixin])
+            await utils1.init()
+            await utils2.init()
+
+            await utils2.updateAcquiringContext({
+                invoiceStatus: CONTEXT_FINISHED_STATUS,
+                invoiceImplicitFeeDistributionSchema: [{
+                    recipient: 'organization',
+                    percent: '5',
+                }],
+                invoiceRecipient: createTestRecipient(),
+            })
+
+            const resident = await utils1.createResident()
+
+            const [marketCategory] = await createTestMarketCategory(utils2.clients.admin)
```

Comment:
utils2.clients.admin - is the easiest sollution, but in real world marketPlace is created by employee